### DISCUSSION TITLE
performance benchmark

### DISCUSSION THREAD

**Discussion by dante01yoon** [vanilla-extract-css/vanilla-extract#876] [Q&A]
Hi team,  I recently highly enjoy using this library and my friends thanks to me introducing them vanilla extract. 

do you have any performance benchmark comparison between css-in-js or tailwind?

**Comment by swernerx** [comment]
Which aspect of performance do you refer to? I guess if there is no client-side execution, then it's zero vs. something. But there is also an effect to the actual build time (tooling). Do you refer to this?

**Reply by dante01yoon** [reply]
parsing times, run time execution when use run time build api like assignInlineVars 

 like your said, also,  whole processing time with styles compare with css in js.
